homeassistant/core.py

```diff
@@ -832,14 +850,16 @@ def has_service(self, domain, service):
         return service.lower() in self._services.get(domain.lower(), [])
 
     def register(self, domain, service, service_func, description=None,
-                 schema=None):
+                 schema=None, state_to_set=None):
```

Comment:
Change it to state and make it a keyword parameter only. In the future I want to move description and schema to be keyword only too (but we should do that in another PR)

```python
def register(self, doman, service, service_func, *, description=None, schema=None, state=None):
```